CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/CaptureSuggestionsViewContainer.swift

```diff
@@ -31,18 +45,19 @@ final class CaptureSuggestionsViewContainer: UIView {
         backgroundColor = UIColor.GiniCapture.systemGray05
         layer.cornerRadius = 16
 
-        titleLabel.adjustsFontForContentSizeCategory = true
         titleLabel.font = configuration.textStyleFonts[.calloutBold]
         titleLabel.textColor = UIColor.GiniCapture.label
 
-        descriptionLabel.adjustsFontForContentSizeCategory = true
         descriptionLabel.font = configuration.textStyleFonts[.subheadline]
         descriptionLabel.textColor = UIColor.GiniCapture.systemGray
     }
 
     func configureContent(with image: UIImage?, title: String, description: String) {
         self.imageView.image = image
         self.titleLabel.text = title
+        self.titleLabel.accessibilityValue = title
```

Comment:
I think there is no need to have accessibilityValue for labels. Then the text is read twice